src/controller.js

```diff
@@ -557,7 +580,7 @@ export default ['$scope', '$element', function ($scope, $element) {
         if (!model.layout.qAppObjectList.qItems) {
           return resolve({ value: '', label: 'No MasterObjects' });
         }
-        // Resolve an array with master objects.
+        // Resolve an array with master objects.        
```

Comment:
Don't like these extra spaces. Won't fail the review because of it though

Reply:
Shall remove them going forward.